pkg/hypperpath/lazypath.go

```diff
@@ -0,0 +1,59 @@
+package hypperpath
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/rancher-sandbox/hypper/pkg/hypperpath/xdg"
+)
+
+const (
+	// CacheHomeEnvVar is the environment variable used by Helm
+	// for the cache directory. When no value is set a default is used.
+	CacheHomeEnvVar = "HYPPER_CACHE_HOME"
+
+	// ConfigHomeEnvVar is the environment variable used by Helm
+	// for the config directory. When no value is set a default is used.
+	ConfigHomeEnvVar = "HYPPER_CONFIG_HOME"
+
+	// DataHomeEnvVar is the environment variable used by Helm
+	// for the data directory. When no value is set a default is used.
+	DataHomeEnvVar = "HYPPER_DATA_HOME"
```

Comment:
Probably, but I do not like the idea, there will be cases where the hypper settings are being overturned by helm stuff consumed through comppositing and this will probably screw up the local helm config of the person running this.